integration/network/network_linux_test.go

```diff
@@ -389,10 +389,17 @@ func checkCtrRoutes(t *testing.T, ctx context.Context, apiClient client.APIClien
 	})
 
 	assert.Check(t, is.Equal(len(routes), expRoutes), "expected %d routes, got %d:\n%s", expRoutes, len(routes), strings.Join(routes, "\n"))
-	defFound := slices.ContainsFunc(routes, func(s string) bool {
-		return strings.Contains(s, expDefRoute)
-	})
-	assert.Check(t, defFound, "default route %q not found:\n%s", expDefRoute, strings.Join(routes, "\n"))
+	if expDefRoute == "" {
+		defFound := slices.ContainsFunc(routes, func(s string) bool {
```

Comment:
Looks like it didn't complain before this, but probably because it's a `_test.go`  - wondering if we should technically have such `//go:build` tags for these as well

(no need to do in this PR; looks like it was already there)